extensions/attention-stream/src/background.ts

```diff
@@ -561,7 +579,7 @@ class GetPingsUploader extends Uploader {
       "youtube-video-recommendations": "id",
       "youtube-ads": "id",
       "tracking-pixel": "id",
-      "attribution": "id"
```

Comment:
Hm, why just this one? :)

Reply:
oh I think that's the default prettier setting (prefer non-quotes and only use quotes if needed i.e. keys with weird characters). I agree it looks kinda funny and would prefer "if one gets quotes all should get quotes" but I also don't want to be explicitly undoing the prettier format every time.

if we decide on the latter as our standard though, I can update my prettier settings?